JSDemos/Demos/Form/Validation/description.md

```diff
@@ -1,10 +1,27 @@
-﻿This demo shows how to&nbsp;validate the [Form][0] editors. [Validation rules][1] are specified for each form item using the **validationRules** property. All editors on&nbsp;the form are combined into a&nbsp;validation group defined by&nbsp;the form&rsquo;s [validationGroup][2] property. 
+﻿This demo shows how to validate the Form editors in simple items. To apply validation rules to an item, specify them in the [validationRules[]](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/) array. Specify [type](/Documentation/ApiReference/UI_Components/dxValidator/Validation_Rules/RequiredRule/#type) and other properties for each rule. 
 
+To submit a Form, add a [Button Item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/ButtonItem/) and set its [useSubmitBehavior](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#useSubmitBehavior) property to **true**. When users click the button, the Form validates all editors that belong to the same group. In this demo, all editors are wrapped into the `customerData` [validationGroup](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#validationGroup). The Form can be submitted to a server only if input validation is successful. Set the [showValidationSummary](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#showValidationSummary) property to **true** to display all validation errors at the bottom of the Form. 
```

Comment:
Taking the next paragraph into account, to submit the form, you need to perform three steps:

1. Wrap our Form in the HTML form.
2. Add a button item.
3. Set useSubmitBehavior.

I believe a list is a good choice here. But then we end up with an ambiguity: when we say "submit the form", do we mean "submit the HTML form" or "submit our Form component". To resolve this ambiguity, I replaced this phrase with "submit form data".

> ... add a [Button Item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/ButtonItem/)... When users click the button

This transition from button item to just button was not really clear, so I made it straight in step 2.

> ... all editors that belong to the same group. In this demo, all editors are wrapped into the `customerData` [validationGroup](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#validationGroup).

I removed the repetition of "all editors" and moved the link to validationGroup to the end of the first sentence since this is where we mention the group for the first time. Also, if you look at the code, the editors aren't actually wrapped in this group - the group is just specified for them using the validationGroup property.

> Set the [showValidationSummary](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#showValidationSummary) property to **true** to display ...

Repetition of "to". I used "enable" to get rid of it. 

```suggestion
To submit form data, do the following:

1. Wrap the Form component in the HTML `<form>` element.

1. Use the [Button Form Item](/Documentation/ApiReference/UI_Components/dxForm/Item_Types/ButtonItem/) to add a button to the form. This button will submit the form data.

1. Enable the button's [useSubmitBehavior](/Documentation/ApiReference/UI_Components/dxButton/Configuration/#useSubmitBehavior) property.

When users click the button, the Form validates editors that belong to the same [validationGroup](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#validationGroup). In this demo, all editors belong to the `customerData` group. Form data can be submitted to a server only if input validation is successful. Enable the [showValidationSummary](/Documentation/ApiReference/UI_Components/dxForm/Configuration/#showValidationSummary) property to display all validation errors at the bottom of the Form.
```